### ISSUE TITLE
Repeatability Question

### ISSUE THREAD

**Issue by kdlin** [OscarEngelbrektson/SyntheticControlMethods#6]
I re-ran your West Germany Reunification example and got different results. In your Jupyter notebook example, you have the following:
pre_rmspe      110.635150
post_rmspe    2239.826549
post/pre        20.245162
Name: 0, dtype: float64

After I re-ran your example, I have the following:
unit          West Germany
pre_rmspe          96.2433
post_rmspe         2246.03
post/pre            23.337

Can you please explain the differences? The result is from this statement: sc.original_data.rmspe_df.iloc[0]. 

Additionally, when I ran the visualization code for in-place placebo plot (i.e., sc.plot(['rmspe ratio'], treated_label="West Germany"), it yielded a chart that is rotated 90 degrees from yours.

**Comment by OscarEngelbrektson** [comment]
Hey @kdlin,

Thanks for all your feedback - I really appreciate it! 

There is some randomness in the starting values for the optimization procedure, which is why it is _possible_ for Synth()  to converge to different solutions for the same dataset (same for DiffSynth) if the starting point were sufficiently different across all optimization iterations. Typically, this is avoidable by increasing n_optim, i.e. the number of different, random starting points for which the optimization is run. Regardless, __I have added random_seed=0__ as a keyword argument in Synth() and DiffSynth(). This will ensure that all results are replicable going forward.

Regarding the "rmspe ratio" plot, I have changed it from a histogram to a barplot. It still displays exactly the same data (Synth.original_data.rsmpe_df). However, plotting it as a sorted barplot allows me to label each bar with the corresponding unit name, which helps with interpretability.

I have updated the [user guide](https://nbviewer.jupyter.org/github/OscarEngelbrektson/SyntheticControlMethods/blob/master/examples/user_guide.ipynb) to reflect these changes in the package.

Best,
Oscar
